libraries/lib-datahandler/src/test/java/org/eclipse/sw360/datahandler/couchdb/AttachmentContentDownloaderTest.java

```diff
@@ -88,6 +87,7 @@ public String call() throws Exception {
                 } catch (ExecutionException e) {
                     Throwable futureException = e.getCause();
                     assertThat(futureException, is(notNullValue()));
+                    assertTrue(futureException instanceof SocketTimeoutException);
```

Comment:
Maybe add `SocketException` or directly use `IOException`

Reply:
I want to avoid `IOExcepioin`, since it contains less information.

`SocketException` occurs on your system?

Reply:
"No route to host (Host unreachable)" should be NoRouteToHostException which extends the SocketException class

Reply:
Oh, now I understand. It depends on the routing of the network interface on the host, which exception gets thrown.